/****************************************************************************
//	Usagi Engine, Copyright Â© Vitei, Inc. 2013
****************************************************************************/

import 'nanopb.proto';
import 'Engine/Graphics/Color.proto';

package usg.Components;

// Turns on debug rendering for any shapes attached to the entity
message DebugRenderData
{
	required uint32 page = 1;
	required Color color = 2;
}
